src/components/modals/DarModal.js

```diff
@@ -71,7 +96,16 @@ const DarModal = (props) => {
       h(ModalDetailRow, {
         label: 'Type of Research',
         detail: processResearchTypes(darDetails.researchType)
-      })
+      }),
+      div({
+        isRendered: requiresManualReview(darDetails),
+        style: Styles.ALERT
+      }, [
+        Alert({
```

Comment:
The alert is great! Just wanted to add that with this addition you can remove the ternary string processing in processResearchTypes and just return type.description.

Current code for reference...
```
const processResearchTypes = (researchTypes) => {
  let researchStatements = '';
  if(!isEmpty(researchTypes)) {
    researchStatements = researchTypes.map(type => {
      return `${type.description} ${type.manualReview ? 'Requires manual review.' : ''}`;
    });
  }
  return researchStatements;
};
```

Reply:
Nice, thank you!